vue-client/src/components/mixins/sidesearch-mixin.vue

```diff
@@ -127,19 +137,15 @@ export default {
       this.fetch(n);
     },
     getItems(keyword) {
-      const searchPhrase = this.getSearchPhrase(keyword);
-      const urlSearchParams = new URLSearchParams({
```

Comment:
Note that `URLSearchParams` can also be passed a sequence of pairs, or have it's ` append(key, value)` method called multiple times with the same key to handle a repeated parameter.

Reply:
Yes, I think it would be preferable to use `URLSearchParams` again (I used the `append` method before but we need to ensure that it applies to all array values). This will be the scope of a later PR.